src/libmongoc/src/mongoc/mongoc-client-side-encryption.c

```diff
@@ -746,9 +816,13 @@ _release_mongocryptd_client (mongoc_client_t *client_encrypted,
 mongoc_collection_t *
 _get_keyvault_coll (mongoc_client_t *client_encrypted)
 {
+   mongoc_write_concern_t *const wc = mongoc_write_concern_new ();
+   mongoc_read_concern_t *const rc = mongoc_read_concern_new ();
```

Comment:
I haven't seen this before, but wouldn't `const` normally come _before_ the struct type?

Also, you later call `mongoc_write_concern_set_w` and `mongoc_read_concern_set_level` on these variables. Those functions don't allow `const` pointers, so perhaps this modifier is ignored and should just be deleted?

Edit: TIL this is making the pointer itself constant (http://bruceblinn.com/linuxinfo/Const.html), so no objections there. This is the first time I've seen this done in libmongoc.